### PR TITLE
english Subtitel für ZAP Chat Video 01-08

### PR DESCRIPTION
**Pull request by 8linkz**
Video URL: 
Video 01: https://www.youtube.com/watch?v=qrAJ32rdi14
Video 02: https://www.youtube.com/watch?v=RCi9W77bGpI
Video 03: https://www.youtube.com/watch?v=cKG4UWa082A
Video 04: https://www.youtube.com/watch?v=9RbmkT9rG5w
Video 05: https://www.youtube.com/watch?v=Rq_d7OLmMfw
Video 06: https://www.youtube.com/watch?v=PnCbIAnauD8
Video 07: https://www.youtube.com/watch?v=19Rptj2be1Y
Video 08: https://www.youtube.com/watch?v=1fcpU54N-mA

Update 18.12.2023:
This file had a lot of hallucinations and I had to make a lot of manual adjustments. At 12:22->12:35 I'm not sure if what's said matches the subtitle.

If anyone else can double check - would be cool.

Update 11.12.2023:
I'm not good with English spelling (commas etc) and Simon sometimes makes long sentences so it's sometimes hard for me to find an "end" to put the point. But that's up to me, so I can't give any guarantees as far as punctuation is concerned. This part would have to be checked by a native speaker or english pro.

I only deleted AI hallucinations, "um" and "uh".

_____________________________________________________
Goal: The goal is that by making the subtitles as good as possible, it makes it easier to translate into other languages - also using AI, for example. This makes the videos accessible to a larger group of people.

How did I create them?
First I downloaded the video and then removed the video track. I transcribed the .mp3 file with the language via Whisper (https://github.com/openai/whisper). Afterwards, I listened to the video to recognize differences and then adjusted them.

Empty parts are ignored by YouTube and I did not remove them (as there is an enumeration here).

### CODE REVIEW

File: ZAP_Chat_04_ZAPit_english.srt [hunk 1]
```diff
@@ -0,0 +1,1076 @@
...
+00:00:11,960 --> 00:00:17,680
+And this is going to be a video about a new feature in the new ZAP release, 2.14, and
+
+3
+00:00:17,680 --> 00:00:19,680
+it's called zapit.
+
+4
+00:00:19,680 --> 00:00:22,640
+And this time I'm actually joined by Yanis.
```
Comment (psiinon) [inline ZAP_Chat_04_ZAPit_english.srt]: Yiannis as per https://www.zaproxy.org/docs/team/yiannis/

File: ZAP_Chat_04_ZAPit_english.srt [hunk 2]
```diff
@@ -0,0 +1,1076 @@
...
+00:08:21,600 --> 00:08:28,440
+technology it's not available because we actually need another add-on for that
+
+145
+00:08:28,440 --> 00:08:38,520
+so let me install that so minus command and then we want to do addoninstall
+
+146
+00:08:38,520 --> 00:08:45,860
+right number of l's. So Simon while you do that tell us a little bit about the wappanalyzer
```
Comment (psiinon) [inline ZAP_Chat_04_ZAPit_english.srt]: wappalyzer : https://www.zaproxy.org/docs/desktop/addons/technology-detection/

File: ZAP_Chat_04_ZAPit_english.srt (hunk 1 above)
Comment (8linkz) [reply ZAP_Chat_04_ZAPit_english.srt]: I went through every file again, replaced it, should fit.

File: ZAP_Chat_04_ZAPit_english.srt (hunk 2 above)
Comment (8linkz) [reply ZAP_Chat_04_ZAPit_english.srt]: I went through every file again, replaced it, should fit.

**Review by psiinon — APPROVED** [review]

**Comment by psiinon** [comment]
Many thanks!

**Comment by psiinon** [comment]
@thc202 @kingthorin either of you want to check this?
I havnt checked the times or that its word for word correct, just that it looks sane 😉

**Comment by thc202** [comment]
Happy to take a look.

IMO this should be added under a directory not to the top level of the repo (e.g. `subtitles/ZAP Chat/`).

File: ZAP_Chat_04_ZAPit_english.srt [hunk 3]
```diff
@@ -0,0 +1,1076 @@
...
+00:01:34,300 --> 00:01:37,740
+So do you want to give us a quick introduction to yourself?
+
+25
+00:01:37,740 --> 00:01:38,880
+Yeah, sure.
+
+26
+00:01:38,880 --> 00:01:41,680
+Just keeping it short, my name is Yiannis Papouasoglou.
```
Comment (thc202) [inline ZAP_Chat_04_ZAPit_english.srt]: Pavlosoglou

File: ZAP_Chat_06_Automation_Introduction_english.srt [hunk 4]
```diff
@@ -0,0 +1,1765 @@
...
+00:07:45,860 --> 00:07:50,900
+getting along. So this is useful it's 
+
+83
+00:07:50,920 --> 00:07:55,740
+give you more information than ZAPit. But depending on the website could take a lot longer.
+
+84
+00:07:56,780 --> 00:08:03,180
+But as Yannis said it uses traditional spider there's no option for the AJAX spider and no
```
Comment (thc202) [inline ZAP_Chat_06_Automation_Introduction_english.srt]: Yiannis

File: ZAP_Chat_06_Automation_Introduction_english.srt [hunk 5]
```diff
@@ -0,0 +1,1765 @@
...
+00:11:40,080 --> 00:11:43,940
+So that's an important point and Simon
+
+119
+00:11:43,960 --> 00:11:49,180
+just real quickly where have we specified in the command line options that this is a baseline scan
+
+120
+00:11:49,180 --> 00:11:56,700
+if you don't mind just showing us that? It was actually there so zapbaseline.py so it's a python
```
Comment (thc202) [inline ZAP_Chat_06_Automation_Introduction_english.srt]: zap-baseline.py

File: ZAP_Chat_06_Automation_Introduction_english.srt [hunk 6]
```diff
@@ -0,0 +1,1765 @@
...
+00:15:06,560 --> 00:15:11,980
+and generally you want to be ready to go from a baseline perspective before you get into the
+
+153
+00:15:11,980 --> 00:15:12,460
+even more advanced scan and then you want to be ready to go from a baseline perspective
+
+154
+00:15:12,480 --> 00:15:16,160
+before you get into the more advanced stuff that simon's about to show us. Yeah and one thing we've
```
Comment (thc202) [inline ZAP_Chat_06_Automation_Introduction_english.srt]: Simon

File: ZAP_Chat_04_ZAPit_english.srt [hunk 7]
```diff
@@ -0,0 +1,1076 @@
...
+00:12:31,940 --> 00:12:39,760
+resources and specific items where wildcards are being used you see the medium CSP content
+
+186
+00:12:39,760 --> 00:12:43,680
+security policy wildcard director they actually need that in order to allow
+
+187
+00:12:43,680 --> 00:12:48,940
+video to be functioning right so going back to simon's quote it's not about taking a medium risk
```
Comment (thc202) [inline ZAP_Chat_04_ZAPit_english.srt]: Simon

File: ZAP_Chat_04_ZAPit_english.srt [hunk 8]
```diff
@@ -0,0 +1,1076 @@
...
+00:15:29,520 --> 00:15:33,700
+comes back but there is something of value that will come back from the response that we're going
+
+220
+00:15:33,700 --> 00:15:40,640
+to see here there we are we see the 404 now what's interesting straight away even though i have a 404,
+
+221
+00:15:41,800 --> 00:15:45,120
+i can see http/3 is beeing used as a technology
```
Comment (thc202) [inline ZAP_Chat_04_ZAPit_english.srt]: being

File: ZAP_Chat_04_ZAPit_english.srt [hunk 9]
```diff
@@ -0,0 +1,1076 @@
...
+00:16:50,520 --> 00:16:55,800
+as well. Yeah absolutely and i have a big b in my bonnet about sql injection still being around
+
+234
+00:16:55,800 --> 00:17:01,720
+because after being in the corporate world for for more than 10 years i set up my second
+
+235
+00:17:01,720 --> 00:17:06,460
+company kyberna and one of the services of course that we had to offer right was pentesting, because
```
Comment (thc202) [inline ZAP_Chat_04_ZAPit_english.srt]: Kiberna

File: ZAP_Chat_06_Automation_Introduction_english.srt [hunk 10]
```diff
@@ -0,0 +1,1765 @@
...
+00:04:46,380 --> 00:04:52,140
+port because I've got tomcat on 8080 and I don't want to clash with that. Minus command because
+
+50
+00:04:52,140 --> 00:05:01,120
+I'm doing this on the command line minus ZAPit and the target url and we'll see very quickly that
+
+51
+00:05:01,120 --> 00:05:08,280
+it's made one request and it got a 200. I haven't actually got the wappalizer add-on installed so
```
Comment (thc202) [inline ZAP_Chat_06_Automation_Introduction_english.srt]: wappalyzer

File: ZAP_Chat_04_ZAPit_english.srt (hunk 8 above)
Comment (kingthorin) [reply ZAP_Chat_04_ZAPit_english.srt]: Is it really 3?

Comment (thc202) [reply ZAP_Chat_04_ZAPit_english.srt]: Yes.

Comment (kingthorin) [reply ZAP_Chat_04_ZAPit_english.srt]: Okay, thanks

File: ZAP_Chat_04_ZAPit_english.srt (hunk 3 above)
Comment (8linkz) [reply ZAP_Chat_04_ZAPit_english.srt]: done

File: ZAP_Chat_04_ZAPit_english.srt (hunk 7 above)
Comment (8linkz) [reply ZAP_Chat_04_ZAPit_english.srt]: done

File: ZAP_Chat_04_ZAPit_english.srt (hunk 8 above)
Comment (8linkz) [reply ZAP_Chat_04_ZAPit_english.srt]: done

File: ZAP_Chat_04_ZAPit_english.srt (hunk 9 above)
Comment (8linkz) [reply ZAP_Chat_04_ZAPit_english.srt]: done

File: ZAP_Chat_06_Automation_Introduction_english.srt (hunk 4 above)
Comment (8linkz) [reply ZAP_Chat_06_Automation_Introduction_english.srt]: done

File: ZAP_Chat_06_Automation_Introduction_english.srt (hunk 5 above)
Comment (8linkz) [reply ZAP_Chat_06_Automation_Introduction_english.srt]: done

File: ZAP_Chat_06_Automation_Introduction_english.srt (hunk 6 above)
Comment (8linkz) [reply ZAP_Chat_06_Automation_Introduction_english.srt]: done

File: ZAP_Chat_06_Automation_Introduction_english.srt (hunk 10 above)
Comment (8linkz) [reply ZAP_Chat_06_Automation_Introduction_english.srt]: done Mh I think I shouldn't review this at 05:30 AM without coffee ~.-

File: ZAP_Chat_04_ZAPit_english.srt (hunk 8 above)
Comment (8linkz) [reply ZAP_Chat_04_ZAPit_english.srt]: grafik

Comment (8linkz) [reply ZAP_Chat_04_ZAPit_english.srt]: in the lower third under "Technology
And google translate http3 into htb3^^

File: ZAP_Chat_07_Automation_Framework_Part_1_english.srt [hunk 11]
```diff
@@ -64,7 +64,7 @@ Great. And I'll move some of the toolbars out of my way so I can see what's goin
 
 17
 00:01:13,640 --> 00:01:18,180
-So what I wanted to show you is start with if we go to documentation.
+So what I wanted to show you is start with if we go to docentation.
```
Comment (thc202) [inline ZAP_Chat_07_Automation_Framework_Part_1_english.srt]: Removed um in other words.

Comment (8linkz) [reply ZAP_Chat_07_Automation_Framework_Part_1_english.srt]: done

Comment (8linkz) [reply ZAP_Chat_07_Automation_Framework_Part_1_english.srt]: so now it says documentation again

Comment (thc202) [reply ZAP_Chat_07_Automation_Framework_Part_1_english.srt]: There were several words not just this one: https://github.com/zaproxy/zap-media-resources/pull/4/commits/4c63d3b0639d12f238ad3db92e0709689d3171e6

Comment (8linkz) [reply ZAP_Chat_07_Automation_Framework_Part_1_english.srt]: ok no idea why this happened now -.- I'll look over it again later

Edit: only "ducumentation" should be affected, I must have clicked wrong...adjusted the rest...

**Comment by 8linkz** [comment]
add .srt file for Video 08.

**Comment by psiinon** [comment]
@8linkz did you see this?

> 
> IMO this should be added under a directory not to the top level of the repo (e.g. `subtitles/ZAP Chat/`).

Also, would be good to open new PRs for new translations otherwise we'll never finish reviewing this one 😉

**Comment by kingthorin** [comment]
Let's not add anything more to this PR (after 8) so that we can get it done and merged.

**Comment by 8linkz** [comment]
> @8linkz did you see this?
> 
> > IMO this should be added under a directory not to the top level of the repo (e.g. `subtitles/ZAP Chat/`).
> 
> Also, would be good to open new PRs for new translations otherwise we'll never finish reviewing this one 😉

Whoever invented the idea of having to create a folder like this (in github) should be tarred and feathered - should be done now.

**Review by psiinon — APPROVED** [review]
LGTM!

File: subtitles/ZAP Chat/ZAP_Chat_07_Automation_Framework_Part_1_english.srt [hunk 12]
```diff
@@ -0,0 +1,1304 @@
...
+00:07:12,600 --> 00:07:13,200
+You'll see them.
+
+96
+00:07:13,200 --> 00:07:15,540
+We have these jobs down here.
+
+97
+00:07:16,320 --> 00:07:22,080
+And at some point I'll try and make these colns better size automatically.
```
Comment (thc202) [inline subtitles/ZAP Chat/ZAP_Chat_07_Automation_Framework_Part_1_english.srt]: `colns` → `columns`

File: subtitles/ZAP Chat/ZAP_Chat_07_Automation_Framework_Part_1_english.srt [hunk 13]
```diff
@@ -0,0 +1,1304 @@
...
+00:07:22,080 --> 00:07:23,220
+Slightly better size.
+
+99
+00:07:23,220 --> 00:07:23,400
+Yeah.
+
+100
+00:07:23,440 --> 00:07:26,200
+So just for argent's sake, Simon, let's just say that.
```
Comment (thc202) [inline subtitles/ZAP Chat/ZAP_Chat_07_Automation_Framework_Part_1_english.srt]: `argent` → `argument`

File: subtitles/ZAP Chat/ZAP_Chat_07_Automation_Framework_Part_1_english.srt [hunk 14]
```diff
@@ -0,0 +1,1304 @@
...
+00:12:43,940 --> 00:12:53,360
+But then we can actually have every, all of the passive scan rules that I have available to me are available here.
+
+165
+00:12:53,360 --> 00:12:54,100
+And we can choose
+
+166
+00:12:54,100 --> 00:12:59,980
+the threshold, which is off, low, medi, or high to go very fine rate.
```
Comment (thc202) [inline subtitles/ZAP Chat/ZAP_Chat_07_Automation_Framework_Part_1_english.srt]: `medi` → `medium`

File: subtitles/ZAP Chat/ZAP_Chat_08_Automation_Framework_Part_2_english.srt [hunk 15]
```diff
@@ -0,0 +1,1116 @@
...
+00:18:36,380 --> 00:18:41,120
+you know broadly speaking then you want to be doing any sort of discovery or 
+
+220
+00:18:41,120 --> 00:18:45,780
+censorship and then you want to be doing the more active scanning
+
+221
+00:18:45,780 --> 00:18:49,200
+type of jobs. Excactly. So
```
Comment (thc202) [inline subtitles/ZAP Chat/ZAP_Chat_08_Automation_Framework_Part_2_english.srt]: `Exactly`

File: subtitles/ZAP Chat/ZAP_Chat_07_Automation_Framework_Part_1_english.srt (hunk 12 above)
Comment (8linkz) [reply subtitles/ZAP Chat/ZAP_Chat_07_Automation_Framework_Part_1_english.srt]: done

File: subtitles/ZAP Chat/ZAP_Chat_07_Automation_Framework_Part_1_english.srt (hunk 13 above)
Comment (8linkz) [reply subtitles/ZAP Chat/ZAP_Chat_07_Automation_Framework_Part_1_english.srt]: done

File: subtitles/ZAP Chat/ZAP_Chat_07_Automation_Framework_Part_1_english.srt (hunk 14 above)
Comment (8linkz) [reply subtitles/ZAP Chat/ZAP_Chat_07_Automation_Framework_Part_1_english.srt]: done

File: subtitles/ZAP Chat/ZAP_Chat_08_Automation_Framework_Part_2_english.srt (hunk 15 above)
Comment (8linkz) [reply subtitles/ZAP Chat/ZAP_Chat_08_Automation_Framework_Part_2_english.srt]: done

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**